e107_plugins/social/languages/English/English_global.php

```diff
@@ -12,5 +12,5 @@
 define("LAN_PLUGIN_SOCIAL_SIGNIN",  "Sign in with:");
 define("LAN_PLUGIN_SOCIAL_XUP_SIGNUP", "Sign-in with your [x] account"); 
 define("LAN_PLUGIN_SOCIAL_XUP_REG", "Register with your [x] account");
-
-
+define("LAN_PLUGIN_SOCIAL_NAME", "Social");
+define("LAN_SOCIAL_WARNING", "Facebook comments requires that you have a facebook App ID. See the 'social login' area in admin-preferences to add one.");
```

Comment:
This warning shouldn't be loaded on every page of the site, but only when required. Better to place it inside English_front.php and make sure the language file is loaded in e_comment.php